docs/simulation_models.rst

```diff
@@ -2,4 +2,205 @@
+        ├── consumers-mass_flow.csv
+        └── consumers-temperature_drop.csv
+
+.. code-block:: python
+
+    import dhnx
+
+    thermal_network = dhnx.network.ThermalNetwork()
+
+    thermal_network.simulate()
+
+
+.. 	figure:: _static/radial_network_details.svg
+   :width: 70 %
+   :alt: radial_network_details.svg
+   :align: left
+
+   Fig. 1: Schematic
+
+
+
+
+
+Underlying Concept
+------------------
+
+.. _simulation_model_table_label:
+
+.. csv-table::
+   :header-rows: 1
+   :delim: ;
+   :file: _static/simulation_models.csv
+
+
+
+The following equations are related to a single pipe.
+
+Hydraulic equations
+~~~~~~~~~~~~~~~~~~~
+
+A pressure difference between two ends of a pipe occurs because of three effects:
+
+* distributed pressure losses along the pipe's inner surface
+* local pressure losses at distinct items,
+* hydrostatic pressure differences because of a difference in height.
+
+All three effects can be captured in this formula:
+
+.. math::
+    \Delta p = \Delta p_{loc} + \Delta p_{dis} + \Delta p_{hydrostatic}
+
+**Distributed pressure losses**
+
+The Darcy-Weissbach-equation describes distributed pressure losses
+:math:`\Delta p_{dis}` inside the pipe:
+
+.. math::
+    \Delta p_{dis} = \lambda \rho \frac{L}{2D} v^2.
+
+
+Together with the flow velocity
+
+.. math::
+    v = \frac{4 \dot{m}}{\rho \pi D^2}
+
+it can be written to:
+
+.. math::
+    \Delta p_{dis} = \lambda \frac{8 L}{\rho \pi^2 D^5} \dot{m}^2,
+
+
+where the darcy friction factor :math:`\lambda = \lambda(Re, \epsilon, D)` depends on the Reynolds number
+:math:`Re:`, the pipe's surface roughness :math:`\epsilon` and the pipe’s inner diameter :math:`D`.
+The Reynolds number is a dimensionless quantity characterizing fluid flows and is defined as follows:
+
+.. math::
+    Re = \frac{Dv\rho}{\mu}.
+
+:math:`\mu` is the dynamic viscosity of water.
+
+In a pipe, flow is laminar if :math:`Re` < 2300 and turbulent if :math:`Re` > 4000.
+In district heating pipes, flow is usually turbulent. The turbulent flow regime can be further
+distinguished into smooth, intermediate and rough regime depending on the pipe's surface roughness.
+
+In Bordin2015 :math:`\lambda` is calculated using the approximation formula:
+
+.. math::
+    \lambda = 0.07 \cdot Re ^{-0.13} \cdot D^{-0.14}.
+
+A more accurate approximation of the Colebrook-White-equation for flow in pipes is given by this
+formula:
+
+.. math::
+    \lambda = \frac{1.325}{(ln(\frac{\epsilon}{3.7D} + \frac{5.74}{Re^{0.9}}))^2}.
+
+**Local pressure losses**
+
+Local pressure losses are losses at junction elements, angles, valves etc. They are described by
+the localized pressure loss coefficient :math:`\zeta`:
+
+.. math::
+    \Delta p_{loc} = \zeta \frac{v^2}{2} \rho
+It is assumed that each fork has a tee and each consumer has a valve installed.
+
+**Hydrostatic pressure difference**
+
+The hydrostatic pressure difference is calculated as follows:
+.. math::
+    \Delta p_{hydrostatic} = - \rho g \Delta h
```

Comment:
```suggestion
The hydrostatic pressure difference is calculated as follows:

.. math::
    \Delta p_{hydrostatic} = - \rho g \Delta h
```

In order to solve problem described in https://github.com/oemof/DHNx/pull/36#issuecomment-693399892